sequence_processing_pipeline/Pipeline.py

```diff
@@ -347,11 +348,41 @@ def get_project_info(self):
 
     @staticmethod
     def is_mapping_file(mapping_file_path):
+        '''
+        Returns true if file is a mapping file.
+        '''
         try:
             Pipeline._validate_mapping_file(mapping_file_path)
             return True
-        except PipelineError:
-            return False
+        except PipelineError as e:
+            # we want to distinguish between a file that is clearly not a
+            # mapping file e.g. sample-file, and a mapping-file that is perhaps
+            # missing a column or has duplicate sample-names.
+            good_messages = ['duplicate sample-names detected:',
+                             'missing columns:']
+            for message in good_messages:
+                if str(e).startswith(message):
+                    return True
```

Comment:
Can this be just messages or explain why good vs. bad?

Reply:
messages sounds good to me.